### DISCUSSION TITLE
When the field is a single file, hope that the image can be displayed in full

### DISCUSSION THREAD

**Discussion by cdwmhcc** [directus/directus#8832] [Feature Requests]
before: 
![image](https://user-images.githubusercontent.com/8030134/137539472-a69fc9bd-40fc-427c-9f3b-b6d3574bfafa.png)

after: 
![image](https://user-images.githubusercontent.com/8030134/137539554-2f0745a6-60d2-4649-a28d-c31c0a421923.png)

**Comment by benhaynes** [comment]
Let's make this an Interface Option, with "contain" as the default.

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
